website/content/docs/connect/gateways/mesh-gateway/service-to-service-traffic-peers.mdx

```diff
@@ -47,7 +47,11 @@ Alternatively, you can also use the CLI to spin up and register a gateway in Con
 
 ### ACL configuration
 
-- If ACLs are enabled, you must add a token granting `service:write` for the gateway's service name and `service:read` for all services in the Enterprise admin partition or OSS datacenter to the gateway's service definition. These permissions authorize the token to route communications for other Consul service mesh services.
+If ACLs are enabled, you must add a token granting `service:write` for the gateway's service name and `service:read` for all services in the Enterprise admin partition or OSS datacenter to the gateway's service definition. 
+These permissions authorize the token to route communications for other Consul service mesh services.
+
+You must also grant `mesh:write` to mesh gateways routing peering traffic in the data plane.
+This permission allows a client certificate to be issued for communication with exported services.
```

Comment:
While the gateway's leaf cert is used for mTLS from the gateway to exported services, it's mostly to be able to terminate TLS and evaluate dynamic routing rules:

```suggestion
This permission allows a leaf certificate to be issued for mesh gateways to terminate TLS sessions for HTTP requests.
```